Racket.gitignore

```diff
@@ -0,0 +1,6 @@
+*~
+\#*
+.\#*
```

Comment:
I'd recommend _not_ ignoring [Emacs lock files](https://emacs.stackexchange.com/questions/57754/how-to-prevent-creating-files-starting-with-lock-files). If they exist it means you have an un-saved file. If you're doing a git operation when you have an unsaved file, it's probably wrong, so seeing the `.#` file in the list of staged files is useful.

(Admittedly this is moot if you use `magit` in Emacs, because it will prompt you about unsaved files.)

Reply:
removed